src/cwe_checker_lib/src/intermediate_representation/term/builder_low_lvl.rs

```diff
@@ -11,6 +12,16 @@
 #[cfg(test)]
 use crate::{expr, intermediate_representation::*, variable};
 
+#[cfg(test)]
+impl<T> Term<T> {
+    /// Sets the field `id` of `Tid`
+    #[cfg(test)]
+    pub fn with_tid_id(mut self, id: String) -> Term<T> {
```

Comment:
to nitpick a little: If the whole `impl`-block is `#[cfg(test)]`, you do not need to add `#[cfg(test)]` to each of the items.